integration/test/chartvalues/main_test.go

```diff
@@ -16,6 +17,7 @@ var (
 	f          *framework.Host
 	helmClient *helmclient.Client
 	l          micrologger.Logger
+	r          *resource.Resource
```

Comment:
Why did `err = r.UpdateResource(cr, templates.UpdatedChartOperatorResourceValues, "stable")` [1] work when this wasn't added before?

[1] https://github.com/giantswarm/chart-operator/pull/107/files#diff-5c729e87953427aa6b0c419816241bc1L51

Reply:
Because you are looking at different tests :P 
Resource got added in the `chartvalues` test and already existed in the `basic` test.

Reply:
:sweat_smile: haha thanks for the pointer.